docs/tutorials/troubleshooting/ip-assignment.md

```diff
@@ -1,2 +1,114 @@
+```
+
+### 3. Check NCP VirtualNetworkInterface Status (NSX-T Networking)
+For NSX-T networking, verify the status of the VirtualNetworkInterface. The expected conditions type should be "Ready," and the IP Addresses should return valid addresses. Use the following command to check the VirtualNetworkInterface status.
+
+**Note** if `vm.spec.networkInterfaces[0].networkName` is empty, then `vnetif_name` should be `<vm_name>-lsp`. Otherwise, `vnetif_name` shoud be `<network_name>-<vm_name>-lsp`.
+
+**Fix** Contact your VI Admin to verify the networking health status.
+
+```console
+$ kubectl describe virtualnetworkinterfaces <vnetif-name> -n <namespace-name>
+Status:
+  Conditions:
+    Status:      True
+    Type:        Ready
+  Ip Addresses:
+    Gateway:      172.26.0.33
+    Ip:           172.26.0.34
+    Subnet Mask:  255.255.255.240
+Events:
+  Type    Reason                        Age   From               Message
+  ----    ------                        ----  ----               -------
+  Normal  SuccessfulRealizeNSXResource  25m   nsx-container-ncp  Successfully realized NSX resource for VirtualNetworkInterface
+```
+
+### 4. Check NetworkInterfacesStatus (VDS Networking)
+For VDS networking, inspect the NetworkInterface status. The conditions type should be "Ready," and the IP Configs should return valid addresses. Use the following command to check the NetworkInterface status.
+
+**Fix** Contact your VI Admin to verify the networking health status.
+
+```console
+$ kubectl describe networkinterface <vm-name> -n <namespace-name>
+Status:
+  Conditions:
+    Last Transition Time:  2023-09-18T19:17:38Z
+    Status:                True
+    Type:                  Ready
+  Ip Configs:
+    Gateway:      192.168.1.1
+    Ip:           192.168.128.42
+    Ip Family:    IPv4
+    Subnet Mask:  255.255.0.0
+  Network ID:     dvportgroup-55
+Events:           <none>
+```
+
+### 5. Bootstrap issues
+When network interfaces issues are ruled out, we will troubleshoot issues caused by [Bootstrap Providers](https://vm-operator.readthedocs.io/en/stable/concepts/workloads/guest/). Here, we'll explore troubleshooting steps for **CloudInit**, **Sysprep**, and **vAppConfig** issues that may affect network connectivity.
+#### a. CloudInit Issues
+For VM deployed using CloudInit bootstrap, if the VM is powered on but doesn't have a valid IPV4 IP assigned, it usually indicates that the CloudInit failed. Follow the steps below to troubleshoot:
+
+1. Check Customization Reconfigure Event: In the vCenter UI, verify if the `Customization Reconfigure` event is present in the VM's events. Its absence suggests a CloudInit failure.
+
+2. Inspect VM ExtraConfig Values:
+  - Ensure that ExtraConfig[guestinfo.metadata] contains metadata generated by the vm-operator, including network configurations and hostname.
+  - Confirm that ExtraConfig[guestinfo.userdata] contains the user-supplied cloud-config data.
```

Comment:
I don't think this is showing nested lists as expected. Try four spaces before the nested items and check the preview doc to confirm the format.